tests/python/test_python_frontend.py

```diff
@@ -4458,6 +4457,53 @@ def fusion_func(fd: FusionDefinition):
         for i in range(num_out):
             self.assertEqual(nvf_out[i].data_ptr(), inputs[0].data_ptr())
 
+    def test_sdpa_fwd(self):
```

Comment:
Do we need a device check for this function? I believe it is restricted to Ampere or newer.

Reply:
Good catch, that is true. Will add it.